superchain/superchain.go

```diff
@@ -127,7 +127,7 @@ func (c ChainConfig) Identifier() string {
 }
 
 type PlasmaConfig struct {
-	DAChallengeAddress *Address `json:"da_challenge_contract_address" yaml:"-"`
+	DAChallengeAddress *Address `json:"da_challenge_contract_address" yaml:"da_challenge_contract_address"`
```

Comment:
From the POV of the downstream packages, this is a required field if `PlasmaConfig` is not nil (in fact I was getting a nil pointer panic in op-node due to the bug of not storing this information). 

I believe we have some checks on this but they weren't sufficient to catch this bug.